pytentiostat/JUAMI_lsvouput.csv

```diff
@@ -0,0 +1,256 @@
+,time,current,voltage
```

Comment:
this looks like a data file which probably shouldn't be in the code directory.  If it is an output file it probably shouldn't be in the repo at all, unless it it is used for testing or as an example in the docs.